packages/helpermodules/update_config.py

```diff
@@ -489,6 +490,7 @@ class UpdateConfig:
         ("openWB/internal_chargepoint/1/data/parent_cp", None),
         ("openWB/optional/et/provider", NO_MODULE),
         ("openWB/optional/int_display/active", False),
+        ("openWB/optional/int_display/detected", True),
```

Comment:
```suggestion
        ("openWB/optional/int_display/active", True),
        ("openWB/optional/int_display/detected", True),
```
Vielleicht sollten hier beide Standardwerte auf Truegesetzt werden. Dann ist automatisch bei vorhandenem Display dieses auch aktiviert.

Reply:
Ist angepasst.